spec/requests/gcs_request_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Gcs', type: :request do
+  describe 'GET /create' do
```

Comment:
should this be a PUT for ccreate?

Reply:
Ah it's supposed to be a `POST /gcs` - this is just the auto-generated tests, and I messed up the `rails g` syntax, I'll update appropriately.